### PR TITLE
Closed Campaign: Only display Winners section if Scholarship

### PR DESCRIPTION
**Pull request by aaronschachter**
@sergii-tkachenko Can you review?

Fixes https://jira.dosomething.org/browse/DS-251

### CODE REVIEW

File: lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.module [hunk 1]
```diff
@@ -91,6 +91,18 @@ function dosomething_campaign_run_preprocess_text_vars(&$vars, $wrapper) {
  *   Node id of the campaign run node.
  */
 function dosomething_campaign_run_preprocess_winners(&$vars, $nid) {
+  // If this campaign has a scholarship:
```
Comment (sergiitk) [inline lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.module]: Maybe

``` php
if (empty($vars['field_scholarship_amount'])) {
  // Don't display the winners section.
  $vars['display_winners'] = FALSE;
  return;
}

// The winners section should be displayed.
$vars['display_winners'] = TRUE;
```

And then just continue?

**Comment by sergiitk** [comment]
Besides, the comment, everything is :+1:

File: lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.module (hunk 1 above)
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.module]: yeah - that's cleaner.  will do

Comment (sergiitk) [inline lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.module]: Great, thanks!

**Pull request merged**